Test/CoreSDK.Test/Shared/Extensibility/MetricAggregatorTest.cs

```diff
@@ -0,0 +1,226 @@
+﻿
+namespace Microsoft.ApplicationInsights.Extensibility
+{
+    using System;
+    using System.Reflection;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using System.Collections.ObjectModel;
+    using System.Collections.Generic;
+    using System.Linq;
+
+    using Microsoft.ApplicationInsights.Channel;
+    using Microsoft.ApplicationInsights.DataContracts;
+    using Microsoft.ApplicationInsights.Extensibility.Implementation;
+    using Microsoft.ApplicationInsights.TestFramework;
+    using Microsoft.VisualStudio.TestTools.UnitTesting;
+    using Assert = Xunit.Assert;
+
+    [TestClass]
+    public class MetricAggregatorTest
```

Comment:
There is a set of tests in MetricAggregatorManager that tests these indirectly, however, I can (and likely should) add MA equality tests.